semantic-source/src/Source/Span.hs

```diff
@@ -78,6 +83,10 @@ instance A.FromJSON Pos where
     [ line, col ] <- A.parseJSON arr
     pure $ Pos line col
 
+instance Lower Pos where
+  lowerBound = Pos 1 1
```

Comment:
Two things: firstly, spans are to be [zero-indexed](https://github.com/github/semantic/issues/263) by default, and secondly, could we get by by omitting these instances and using `(Span.point (Span.Pos 1 1))` at the callsite instead? It's not clear, at a given callsite, what the semantics of `lowerBound` are.

Reply:
@patrickt thanks for pointing that out, I've updated this in https://github.com/github/semantic/pull/518/commits/78e94e37f7254ee204616cf9ccfb6a3e20a9df11